examples/cloud-run-and-hosting/config/server.js

```diff
@@ -0,0 +1,16 @@
+module.exports = ({ env }) => {
+  const config = {
+    host: env('HOST', '0.0.0.0'),
+    port: env.int('PORT', 8081)
+  };
+
+  if (process.env.NODE_ENV === 'production') {
+    config.url = '/api';
```

Comment:
Need to double-check that this `/api` path works

Reply:
I can't get this to work with Strapi. It seems to only work as intended when this is an absolute URL, which means we need to hard-code the URL to the Cloud Run service.